grafana-plugin/src/pages/incident/incident.styles.ts

```diff
@@ -0,0 +1,220 @@
+import { css } from '@emotion/css';
```

Comment:
nit: seems our convention for file naming is `Incident.styles.ts` (uppercase)

Reply:
Didn't notice it, good catch